pkgs/development/libraries/libvorbis/default.nix

```diff
@@ -5,7 +5,8 @@ stdenv.mkDerivation rec {
   version = "1.3.7";
 
   src = fetchurl {
-    url = "http://downloads.xiph.org/releases/vorbis/${pname}-${version}.tar.xz";
+    url =
+      "http://downloads.xiph.org/releases/vorbis/${pname}-${version}.tar.xz";
```

Comment:
Revert this formatting change, but might as well update the url to `https` while you're at it.